template/package.json

```diff
@@ -35,6 +35,7 @@
         "preact-router": "^3.2.1"
     },
     "devDependencies": {
+        "@types/react": "^16.9.19",
```

Comment:
this seems unrelated, also would prefer not having conflicts, I know the current situation isn't ideal

Reply:
https://github.com/preactjs/preact/issues/2222

I hope this will provide a viable alternative

Reply:
I've replaced `@types/react` with a declaration file that exports the Preact JSX namespace, hopefully this is sufficient? Looks to be the simplest way to solve the issue while not adding too much extra to the template.

Reply:
We can expand it later, or maybe compat can somehow include it